content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/android.md

```diff
@@ -179,26 +179,76 @@ In addition to [tracking resources automatically][6], you can also track specifi
 
 To track specific errors, notify the monitor when an error occurs with the message, source, exception, and additional attributes. Refer to the [Error Attributes documentation][9].
 
-   ```kotlin
-      GlobalRumMonitor.get().addError(message, source, throwable, attributes)
-   ```
+```kotlin
+   GlobalRumMonitor.get().addError(message, source, throwable, attributes)
+```
 
-## Track custom global attributes
+### Add user properties
+
+You can use the `addUserProperties` API to append extra user properties to previously set properties.
+
+```kotlin
+fun addUserProperties(extraInfo: Map<String, Any?>, sdkCore: SdkCore = getInstance()) {
+    sdkCore.addUserProperties(extraInfo)
+}
+```
+
+## Event and data management
+
+The Android SDK first stores events and only uploads events when the [intake specifications][11] conditions are met.
+
+### Clear all data
+
+You have the option of deleting all unsent data stored by the SDK with the `clearAllData` API.
+
+```kotlin
+fun clearAllData(sdkCore: SdkCore = getInstance()) {
+    sdkCore.clearAllData()
+}
+```
+
+### Stop data collection
+
+You can use the `StopInstance` API to stop the SDK instance assigned to the given name (or the default instance if the name is null) from collecting and uploading data further.
+
+```kotlin
+   fun stopInstance(instanceName: String? = null) {
+       synchronized(registry) {
+           val instance = registry.unregister(instanceName)
+           (instance as? DatadogCore)?.stop()
+       }
+   }
+```
 
-In addition to the [default RUM attributes][3] captured by the RUM Android SDK automatically, you can choose to add additional contextual information, such as custom attributes, to your RUM events to enrich your observability within Datadog. Custom attributes allow you to slice and dice information about observed user behavior (such as cart value, merchant tier, or ad campaign) with code-level information (such as backend services, session timeline, error logs, and network health).
+### Control event buildup
 
-### Track User Sessions
+Many operations, such as data processing and event input/output, are queued in background threads. To handle edge cases where the queue has grown so much that there could be delayed processing, high memory usage, or Application Not Responding (ANR) errors.
```

Comment:
Would it make sense to combine these two sentences? e.g., "Many operations, such as data processing and event input/output, are queued in background threads in order to handle edge cases where..."